extension/data/modules/usernotes.js

```diff
@@ -1045,6 +1045,10 @@ function usernotes () {
                     <div class="tb-un-info">
                         <span class="tb-info">There are ${userCount} users with ${noteCount} notes.</span>
                         <br> <input id="tb-unote-user-search" type="text" class="tb-input" placeholder="search for user"> <input id="tb-unote-contents-search" type="text" class="tb-input" placeholder="search for note contents">
+                        <select name="tb-un-filter" id="tb-un-filter" class="selector tb-action-button">
+                            <option value="all" default>All</option>
+                            ${colors.map(op => `<option value=${op.key}>${op.text}</option>`).join('')}
```

Comment:
We unfortunately aren't super consistent about this in the rest of the code right now, but any time user input is put into HTML, it should be run through `TBHelpers.htmlEncode()` first.

```suggestion
                            ${colors.map(op => `<option value=${op.key}>${TBHelpers.htmlEncode(op.text)}</option>`).join('')}
```